code/__HELPERS/global_lists.dm

```diff
@@ -137,16 +137,25 @@ GLOBAL_LIST_EMPTY(randomized_pill_icons)
 	for(var/path in subtypesof(/datum/reagent))
 		var/datum/reagent/D = new path()
 		GLOB.chemical_reagents_list[path] = D
+		GLOB.chemical_required_reagents[path] = list()
```

Comment:
Is this needed?

Reply:
This was originally to improve how easier it was to look up cood and its required reagents, we originally planned for some harder mechanics for cloning around this, but never got around to actually implementing it.

I left it in because it was useful to Hugh at the time.

Reply:
But it's currently just creating empty lists?

Reply:
Originally my thought was, this would set the default for every reagent in the case that it doesn't have a reaction the list would be empty.

However, if they just don't appear we can also assert they don't have reactions